app/Metadata/DiskUsage.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Metadata;
+
+//use RecursiveDirectoryIterator;
+//use RecursiveIteratorIterator;
+
+class DiskUsage
+{
+	/**
+	 * Returns true if the current system is windows.
+	 *
+	 * @return bool
+	 */
+	public function is_win()
+	{
+		$os = strtoupper(substr(PHP_OS, 0, 3));
+
+		return $os === 'WIN';
+	}
+
+	/**
+	 * From https://www.php.net/manual/en/function.disk-total-space.php.
+	 *
+	 * @param $bytes
+	 *
+	 * @return string
+	 */
+	public function getSymbolByQuantity($bytes)
+	{
+		$symbols = array(
+			'B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB',
+		);
+		$exp = intval(floor(log($bytes) / log(1024)));
+
+		return sprintf('%.2f %s', ($bytes / pow(1024, floor($exp))), $symbols[$exp]);
```

Comment:
I think that `floor` is not needed here since you're already doing `intval(floor())` above.

Reply:
gotcha.